lib/shared/bucketing-assembly-script/assembly/bucketing/bucketing.ts

```diff
@@ -222,7 +222,10 @@ export function _generateBucketedConfig(
             variation_id,
             variation.name,
             variation.key,
-            null
+            null,
+            feature.name,
+            feature.description,
```

Comment:
Why would we want these server-side? this is going to be expensive in bandwidth to send down to configs.

Reply:
ah.. right, Bucketing-Assembly-Script is only used Server Side so we wouldn't need this since the user will never be able to enrol into a Beta / Opt-In feature program on the Server Side, only on client side

Reply:
Even client-side we probably shouldn't be including that in all configs

Reply:
also exposing internal data like that in our config responses client-side probably is a bad idea.

Reply:
would it make sense to do it the same way as the config generation, conditional checking based on if the `feature.settings.optInEnabled` is true, then add these keys? 🤔